processor/src/chiplets/ace/instruction.rs

```diff
@@ -0,0 +1,48 @@
+use vm_core::Felt;
+
+/// Number of bits used to represent the ID of a node in the evaluation graph.
+/// Define as 30 bits to ensure two indices and the operation can be encoded in a single `Felt`
+pub const ID_BITS: u64 = 30;
+
+/// Maximum allowed ID, also equal to the mask extracting an ID from the lower 30 bits of a
+/// `uint`.
+pub const MAX_ID: u32 = (1 << ID_BITS) - 1;
+
+/// Arithmetic operation applied to two nodes in the evaluation graph.
```

Comment:
nit: Technically, shouldn't this say "applied to two wires" instead of "nodes"? Since the result of an operation is encoded in a wire.

Not so much trying to be picky than making sure I understand correctly how everything fits together (and making sure comments don't cause confusion for future readers).

Reply:
Yeah, that's a bit confusing, changed it to `wire`